doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note (AIX):* This function only works for AIX 7.1 and newer. It can still be
+called on older versions but will return `UV_ENOSYS`.
+
```

Comment:
Never mind me. @jasnell's is fine with perhaps another `AIX` stuck before `7.1` as @richardlau suggests.